### ISSUE TITLE
Monthly itinerary and empty lines adding don't work.

### ISSUE THREAD

**Issue by Marc176353** [klimeryk/recalendar.js#5]
Tried adding a couple of itineraries and empty lines to the month overview - doesn't work. Instead of itinerary - empty lines, while I cannot add empty lines specifically. Same in Week retrospective. Google Chrome Browser.

**Comment by FlorianKempenich** [comment]
Same here. I tried on Chrome, Firefox and Safari. It works for the Days however

**Comment by FlorianKempenich** [comment]
Oh @klimeryk it seems we are synchronized 😂
Glad to see you fixed it already. When will it be live? 

(amazing software btw 😃)

**Comment by klimeryk** [comment]
Haha, jinx!

Yup, that's my bad. A small error crept in as I was refactoring that code (see https://github.com/klimeryk/recalendar.js/commit/657d27bce5422ffa518960fddd363334bd868ec0). I didn't catch it earlier, cause you need to change the itinerary to trigger it 🤦‍♂️  It should be fixed now 🙇‍♂️

**Comment by Marc176353** [comment]
> Haha, jinx!
> 
> Yup, that's my bad. A small error crept in as I was refactoring that code (see [657d27b](https://github.com/klimeryk/recalendar.js/commit/657d27bce5422ffa518960fddd363334bd868ec0)). I didn't catch it earlier, cause you need to change the itinerary to trigger it 🤦‍♂️ It should be fixed now 🙇‍♂️
> 
> Edit: or maybe not. I think the deploy didn't go through. Let me try again.
> 
> Edit2: Yup, fixed now 😌
> 
> Thanks for the reports!

Thank you! It's works now! It's so cool!
